ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +16,6 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+class TestNSError: NSError {
```

Comment:
Oh wow, I'm surprised the docs say that. You can provide a localized description by setting the `NSLocalizedDescriptionKey` key in the user info. No need to subclass! :)